Sources/NIOCore/ChannelHandlers.swift

```diff
@@ -237,19 +248,18 @@ public final class IdleStateHandler: ChannelDuplexHandler, RemovableChannelHandl
 
         let writePromise = promise ?? context.eventLoop.makePromise()
         writePromise.futureResult.whenComplete { (_: Result<Void, Error>) in
-            self.lastWriteCompleteTime = .now()
+            self.lastWriteCompleteTime = self.now()
         }
         context.write(data, promise: writePromise)
     }
 
     private func shouldReschedule(_ context: ChannelHandlerContext) -> Bool {
-        if context.channel.isActive {
-            return true
-        }
-        return false
+        return context.channel.isActive
     }
 
-    private func makeReadTimeoutTask(_ context: ChannelHandlerContext, _ timeout: TimeAmount) -> (() -> Void) {
+    private typealias TimeoutTask = () -> Void
```

Comment:
It was quite hard to follow some of these functions, so I added a couple of type aliases. Let me know if I should remove these @Lukasa